Library/Formula/sqlitebrowser.rb

```diff
@@ -14,12 +12,18 @@ class Sqlitebrowser < Formula
     sha1 "b633087ab28885f058bc5bcf24ca834828d1c894" => :mountain_lion
   end
 
-  depends_on 'qt'
-  depends_on 'cmake' => :build
-  depends_on 'sqlite' => 'with-functions'
+  depends_on "qt"
+  depends_on "cmake" => :build
+  depends_on "sqlite" => "with-functions"
+  depends_on "sqlcipher" => :optional
 
   def install
     system "cmake", ".", *std_cmake_args
-    system 'make install'
+    system "make", "install"
+  end
+
+  test do
+    shell_output("#{bin}/sqlitebrowser -h")
+    assert_equal 0, $?.exitstatus
```

Comment:
Don't need this assert as `shell_output` will assert for us. Also, you probably want to either use `system` which will assert the exit status or use `assert_match` to match some text from the `shell_output`. Thanks for adding a test though!